src/Document/Editor.js

```diff
@@ -0,0 +1,123 @@
+//import FindTitle from "./FindTitle.js"; 구현 실패
+import { request } from "../../library/api.js";
+import { changeRootTitle } from "../../library/titleChanger.js";
+import { markdownToHtml, htmlToMarkdown } from "../../library/convert.js";
+export default function Editor({ $page, initialState }) {
+  this.state = initialState;
+
+  const handleKeyUp = async (e) => {
+    if (e.target.className === "editor-title") {
+      handleTitle(e.target.value);
+    } else if (e.target.className === "editor-content") {
+      handleContent(e.target);
+    }
+  };
+
+  const handleTitle = async (e) => {
+    const nextState = {
+      ...this.state,
+      title: e,
+    };
+    if (e === "") {
+      nextState.title = "제목없음";
+    }
```

Comment:
아래와 같이 작성하면 코드 보기가 한결 좋아질거 같네요!

```suggestion
    const nextState = {
      ...this.state,
      title: e || "제목없음",
    };
```